dbms/src/AggregateFunctions/IAggregateFunction.h

```diff
@@ -149,6 +151,11 @@ class IAggregateFunction
     virtual void addBatchArray(
         size_t batch_size, AggregateDataPtr * places, size_t place_offset, const IColumn ** columns, const UInt64 * offsets, Arena * arena) const = 0;
 
+    virtual AggregateFunctionPtr getOwnNullAdapter(const AggregateFunctionPtr & /*nested_function*/, const DataTypes & /*arguments*/, const Array & /*params*/) const
```

Comment:
Missing a comment.

Reply:
@alexey-milovidov 
I wrote a comment of `getOwnNullAdapter()` containing short description for the function and arguments.